composer.json

```
@@ -57,33 +61,16 @@
         },
         "symfony": {
             "require": "^6.4"
-        }
+        },
+        "public-dir": "vendor/sylius/test-application/public"
+
     },
     "autoload": {
         "psr-4": {
             "Acme\\SyliusExamplePlugin\\": "src/",
             "Tests\\Acme\\SyliusExamplePlugin\\": "tests/"
         }
     },
-    "autoload-dev": {
-        "classmap": [
-            "tests/Application/Kernel.php"
-        ]
-    },
-    "scripts": {
-        "post-install-cmd": [
-            "php bin/create_node_symlink.php"
-        ],
-        "post-update-cmd": [
-            "php bin/create_node_symlink.php"
-        ],
-        "post-create-project-cmd": [
-            "php bin/create_node_symlink.php"
-        ],
-        "auto-scripts": {
-            "cache:clear": "symfony-cmd",
-            "assets:install %PUBLIC_DIR%": "symfony-cmd",
-            "COMPOSER_AUDIT_ABANDONED=ignore composer audit": "script"
-        }
-    }
+    "minimum-stability": "dev",
```

```suggestion
    "minimum-stability": "alpha",
```
Ultimately this node should be removed, We need to release a stable test app for that though